### PR TITLE
git-prune: add page

### PR DESCRIPTION
**Pull request by sanehab**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by sanehab** [comment]
I'm interested in joining tldr to contribute more than just adding new pages, i can help with git and linux commands.

**Comment by navarroaxel** [comment]
> I'm interested in joining tldr to contribute more than just adding new pages, i can help with git and linux commands.

@sanehab You should merge 5 non-trivial PRs to be allowed to [become a collaborator](https://github.com/tldr-pages/tldr/blob/master/COMMUNITY-ROLES.md#when-to-change-roles). :muscle:

File: pages/common/git-prune.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+> Git command for pruning all unreachable objects from the object database.
+> This command is often not used directely but as an internal command that is used by git gc.
+> More information: <https://git-scm.com/docs/git-prune>.
+
+- Report what would be removed by git prune without removing it:
+
+`git prune --dry-run`
+
+- Prune unreachable objects and show what has been pruned to stdout:
```
Comment (navarroaxel) [inline pages/common/git-prune.md:11]:
```suggestion
- Prune unreachable objects and display what has been pruned to `stdout`:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Thanks @sanehab, just 1 comment.

**Review by navarroaxel — APPROVED** [review]

**Review by owenvoke — APPROVED** [review]
Nice job, thanks @sanehab! 👏🏻

**Pull request merged**